fetch.bs

```diff
@@ -1187,16 +1187,19 @@ or "<code>worker</code>".
    <li>
     <p>When <var>read</var> is fulfilled with an object whose <code>done</code>
     property is false and whose <code>value</code> property is a
-    <code>Uint8Array</code> object, run these substeps:
+    <code>Uint8Array</code> object, run these substeps <a>in parallel</a>:
```

Comment:
I use the term "step" for outer steps (e.g., "Let _body_ be _request_'s body") and "substep" for inner steps (e.g., "Transmit _bs_"). The below "Run the above step again." relies on that.